gradle/libs.versions.toml

```diff
@@ -7,9 +7,9 @@ hamcrest = { strictly = '2.2' }
 zalando = { strictly = '0.27.1' }
 aws-lambda-java-core = { strictly = '1.2.3' }
 google-guava = { strictly = '33.0.0-jre' }
-karate-core = { strictly = '1.4.1' }
+karate-core = { strictly = '1.3.1' }
```

Comment:
Deprecating for a reason?

Reply:
Version 1.4.1 did not recognize the feature files as valid in intellij and gave warnings and did not give me the chance to run the feature files individually in IDE